packages/actions/test/unit/authentication.test.ts

```diff
@@ -85,10 +90,43 @@ describe("Authentication", () => {
             )
         })
 
+        it("should not be possible to authenticate if the user has been disabled from the Authentication service by coordinator", async () => {
```

Comment:
These should not be within the describe block `getCurrentFirebaseAuthUser`.  Let's create another describe block and put them there together.

Reply:
Reflected in bac60fc67a2758a0cbf85591b7d9d98c9b638387